business/dashboards.go

```diff
@@ -222,12 +222,18 @@ func (in *DashboardsService) GetDashboard(authInfo *api.AuthInfo, params models.
 	}()
 
 	wg.Wait()
+	// A dashboard can define the rows used, if not defined, by default it will use 2 rows
+	rows := dashboard.Rows
+	if rows == 0 {
```

Comment:
perhaps this should be < 1 instead of 0

Reply:
Yes, agree, just in case, anyway in the UI is well handled:

```typescript
const rows = this.props.dashboard.rows > 0 ? this.props.dashboard.rows : 2;
```